src/HaskellWorks/Data/Network/Ip.hs

```diff
@@ -55,10 +56,18 @@ lastIpv4Address :: Z.Ipv4Block -> Z.Ipv4Address
 lastIpv4Address b@(Z.Ipv4Block (Z.Ipv4Address base) _) = Z.Ipv4Address (base + fromIntegral (blockSize b) - 1)
 
 textToMaybeIpv4Address :: T.Text -> Maybe Z.Ipv4Address
-textToMaybeIpv4Address t = join $ AP.maybeResult <$> AP.parseWith (return mempty) APT.ipv4Address t
+textToMaybeIpv4Address t = AP.maybeResult =<< AP.parseWith (return mempty) APT.ipv4Address t
 
 ipv4AddressToString :: Z.Ipv4Address -> String
 ipv4AddressToString = show
 
 ipv4AddressToText :: Z.Ipv4Address -> T.Text
 ipv4AddressToText = T.pack . ipv4AddressToString
+
+ipv4AddressToWords :: Z.Ipv4Address -> (Word8, Word8, Word8, Word8)
```

Comment:
nit: `ipv4AddressToWord8s`?

Reply:
I think you can see that in the type, so it's technically superfluous info